packages/core-mobile/app/vmModule/ApprovalController/ApprovalController.ts

```diff
@@ -42,13 +43,15 @@ class ApprovalController implements VmModuleApprovalController {
         account,
         maxFeePerGas,
         maxPriorityFeePerGas,
-        overrideData
+        overrideData,
+        isTestnet
```

Comment:
let's remove isTestnet param here also and just use network.isTestnet

Reply:
removed